components/BottomNavigation/README.md

```diff
@@ -181,3 +181,56 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+## Accessibility {#a11y}
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+### `-accessibilityLabel`
+The label will be the title of the UITabBarItem. Currently you can't set this to a custom value.
```

Comment:
Is this a bug or an intentional feature? Is there a backlog issue if it's a bug?

Reply:
I believe it's an intentional feature. The standard accessibility value is `tab` but if there is a title on the tabBarItem then it will be `${title} tab`.